sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -0,0 +1,183 @@
+
+### Adding the package to your product
+
+[//]: # ({x-version-update-start;com.azure:azure-ai-formrecognizer;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-ai-formrecognizer</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Create a Form Recognizer resource
+Form Recognizer supports both [multi-service and single-service access][service_access]. Create a Cognitive Services 
+resource if you plan to access multiple cognitive services under a single endpoint/key. For Form Recognizer access only,
+create a Form Recognizer resource.
+
+You can create either resource using the 
+
+**Option 1:** [Azure Portal][create_new_resource] 
+
+**Option 2:** [Azure CLI][azure_cli]
+
+Below is an example of how you can create a Form Recognizer resource using the CLI:
+
+```bash
+# Create a new resource group to hold the Form Recognizer resource -
+# if using an existing resource group, skip this step
+az group create --name my-resource-group --location westus2
+```
+
+```bash
+# Create Form Recognizer
+az cognitiveservices account create \
+    --name text-analytics-resource \
+    --resource-group my-resource-group \
+    --kind TextAnalytics \
+    --sku F0 \
+    --location westus2 \
+    --yes
+```
+### Authenticate the client
+In order to interact with the Form Recognizer service, you will need to create an instance of the `FormRecognizerClient` 
+class. You will need an **endpoint** and either an **API key** to instantiate a client 
+object. And they can be found in the [Azure Portal][azure_portal] under the "Quickstart" in your created
+Form Recognizer resource. See the full details regarding [authentication][authentication] of Cognitive Services.
+
+#### Get credentials
+The authentication credential may be provided as the API key to your resource or as a token from Azure Active Directory.
+
+##### **Option 1**: Create FormRecognizerClient with API Key Credential
+To use an [API key][api_key], provide the key as a string. This can be found in the [Azure Portal][azure_portal] 
+   under the "Quickstart" section or by running the following Azure CLI command:
+
+```bash
+az cognitiveservices account keys list --resource-group <your-resource-group-name> --name <your-resource-name>
+```
+Use the API key as the credential parameter to authenticate the client:
+<!-- embedme ./src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java#L34-L37 -->
+```java
+FormRecognizerClient formRecognizerClient = new FormRecognizerClientBuilder()
+    .apiKey(new FormRecognizerApiKeyCredential("{api_key}"))
+    .endpoint("{endpoint}")
+    .buildClient();
+```
+The Azure Form Recognizer client library provides a way to **rotate the existing API key**.
+
+<!-- embedme ./src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java#L54-60 -->
+```java
+FormRecognizerApiKeyCredential credential = new FormRecognizerApiKeyCredential("{api_key}");
```

Comment:
Needs to be updated since `FormRecognizerApiKeyCredential` is no longer used in this API.